packages/editor/src/components/dialogs/CreatePrefabPanelDialog.tsx

```diff
@@ -92,19 +90,21 @@ export default function CreatePrefabPanel({ entity }: { entity: Entity }) {
       const resource = resources.data[0]
       const tags = [...prefabTag.value]
       await Engine.instance.api.service(staticResourcePath).patch(resource.id, { tags: tags })
-      PopoverState.hidePopupover()
-      defaultPrefabFolder.set('assets/custom-prefabs')
-      prefabName.set('prefab')
-      prefabTag.set([])
+
       removeEntity(prefabEntity)
-      const { entityUUID } = EditorControlFunctions.createObjectFromSceneElement(
+      await EditorControlFunctions.createPrefabObjectFromSceneElement(
         [
           { name: ModelComponent.jsonID, props: { src: fileURL } },
           { name: TransformComponent.jsonID, props: { position, rotation, scale } }
         ],
-        parentEntity
+        entity,
```

Comment:
should this be parentEntity?

Reply:
I made a new function for create prefab object from scene, here the entity means the original entity which will be unload in the scene and remove the node in snapshot.

Reply:
so its intended, gotcha